Tests/LibSQL/TestSqlValueAndTuple.cpp

```diff
@@ -1240,7 +1240,7 @@ TEST_CASE(shift_right)
         SQL::Value value2 { 2 };
 
         auto result = value1.shift_right(value2);
-        EXPECT(!result.is_error());
+        MUST(result);
```

Comment:
This doesn't work the way you are expecting, because MUST() will call release_value() on the underlying error type and return it from that expression.

That's why all these tests are failing :^). you would have to assign the result of must to a variable in order to compare it.

Reply:
Indeed!

Given that we already call `.value()` anyway, which in turn does `VERIFY(has<T>());`, I'd like to just remove that line entirely.

"But Ben!" you might say, "If you do that, it will crash in case of an error!"
Yes, but that's the current behavior anyway, so there's no loss in testing power.